packages/core/src/components/Typography/FontSize/EditFontSizeModal.tsx

```diff
@@ -60,15 +60,28 @@ export function EditFontSizeModal({
   )
 
   const [error, setError] = useState<string | null>(null)
+  const variantNameRef = useRef() as MutableRefObject<HTMLInputElement>
+  const variantValueRef = useRef() as MutableRefObject<HTMLInputElement>
 
   const handleSave = () => {
     setError(null)
-    if (variantName === '') {
+    if (variantName === '' || variantValue === '') {
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+      variantName === ''
+        ? variantNameRef.current.focus()
+        : variantValueRef.current.focus()
```

Comment:
Can we make this more readable? I think we can do the variantValue check in a different if condition.